fireblocks_sdk/sdk.py

```diff
@@ -58,9 +58,10 @@ def get_nft(self, id: str):
 
         return self._get_request(url)
 
-    def get_nfts(self, ids: List[str], page_cursor: str = '', page_size: int = 100):
+    def get_nfts(self, ids: List[str], page_cursor: str = '', page_size: int = 100, sort: str = None, order: str = None):
```

Comment:
what are the possible params for sort/order? is it a dynamic string? 
If that is a closed list, maybe create a ENUM for it?

Reply:
ASC and DESC for order and for sort we have one possible value for now, but that will change in few days I think

We can use enum